tasks/fix-cat2.yml

```diff
@@ -897,6 +897,7 @@
         regexp: '([A-Z]+ = .*)(sha256\+sha512)(.*)'
         replace: '\1sha512\3'
       register: rhel_07_021620_sha256_512_result
+      notify: "{{ rhel7stig_aide_handler }}"
       failed_when:
         - rhel_07_021620_sha256_512_result | failed
         - rhel_07_021620_sha256_512_result.rc != 257
```

Comment:
While we're making changes here, we can get rid of the custom failed_when since this is now tied to the PRELIM task.  Could be a separate PR, though.

Reply:
There isn't a stat task for /etc/aide.conf in prelim yet so the failed_when is technically still needed for now. I'll add that in a later PR I think.

Reply:
later PR is fine.  /etc/aide.conf is provided by the aide package, so it's fine to error out if it's not there.